sdk/iptables/iptables.go

```diff
@@ -0,0 +1,158 @@
+package iptables
+
+import (
+	"errors"
+	"fmt"
+	"os/exec"
+	"strconv"
+)
+
+const (
+	// Chain to intercept inbound traffic
+	ProxyInboundChain = "PROXY_INBOUND"
+
+	// Chain to redirect inbound traffic to the proxy
+	ProxyInboundRedirectChain = "PROXY_IN_REDIRECT"
+
+	// Chain to intercept outbound traffic
+	ProxyOutputChain = "PROXY_OUTPUT"
+
+	// Chain to redirect outbound traffic to the proxy
+	ProxyOutputRedirectChain = "PROXY_REDIRECT"
```

Comment:
This could be a place to add the default inbound and outbound ports.

For inbound it would be 21000, since that's the default for Connect proxies.